_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,271 @@
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+Some examples use a fixed font size to demonstrate specific aspects of the rule (notably in order to provide a [computed][] value for the property). This is in general not a very good practice and should be avoided.
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `p` element has a [computed][] `letter-spacing` of `2.4px` (assuming a default `medium` font size of `16px`) which is above the recommended minimum, thus it matches the **above minimum** condition.
+
+```html
+<p style="letter-spacing: 0.15em !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 2
+
+This `p` element has a [computed][] [letter-spacing][] of `3px` specified via the style attribute, which is equal to the recommended minimum given the specified font size is 25 pixels, thus it matches the **above minimum** condition.
+
+```html
+<style>
+	p {
+		font-size: 25px;
+	}
+</style>
+
+<p style="letter-spacing: 3px !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 3
+
+This `p` element has two [declared][] values for its `letter-spacing` property. The latest wins the [cascade sort][]. It has a value of `0.15em`, more than 0.12 times the font size and therefore matches the **above minimum** condition.
+
+```html
+<p style="letter-spacing: 0.1em !important; letter-spacing: 0.15em !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 4
+
+This `p` element has two [declared][] values for its `letter-spacing` property. The one which is [important][] wins the [cascade sort][]. It has a value of `0.15em`, more than 0.12 times the font size and therefore matches the **above minimum** condition.
+
+```html
+<p style="letter-spacing: 0.15em !important; letter-spacing: 0.1em">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 5
+
+This `p` element has a not [important][] [computed][] `letter-spacing`, thus it matches the **not important** condition. Even though the value is too small, styles with [author origin][] declared by assistive technologies may win the [cascade sort][] and override it, thus this may satisfy [Success Criterion 1.4.12 Text Spacing][sc1412] and does not fail this rule. This is nonetheless bad practice and sufficient spacing should be used.
```

Comment:
Kinda think this is enough. The rest is explained in other parts of the rule. This just feels a little repetitive. I would also suggest we make this the first example. The "default" case should but that it isn't important.

```suggestion
This `p` element has a not [important][] [computed][] `letter-spacing`.
```

Reply:
I'm always a bit uncomfortable giving bad practice in a Passed Example without an explicit warning… Tried to terse it a bit, let me know if that's OK.